atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -306,6 +307,17 @@ def get_clusters_client_config_path(workflow, fallback=NO_FALLBACK):
     return os.path.join(client_config_dir, 'osbs.conf')
 
 
+def get_platform_to_goarch_mapping(workflow,
```

Comment:
It would be great to use this in `pull_base_image` too, if possible.

Reply:
Added.